pages/content/amp-dev/documentation/guides-and-tutorials/learn/email_fundamentals.md

```diff
@@ -49,7 +49,8 @@ Since the AMP for Email library is a subset of the AMP HTML library, many of the
 
 #### Images
 
-To paint the page effectively, all `<img>` tags are replaced with [`<amp-img>`](https://amp.dev/documentation/components/reference/amp-img.html?format=email). The `<amp-img>` tag requires a defined width and height and supports [AMP's layout system](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/index.html?format=email).
+To paint the page effectively, all `<img>` tags are replaced with [`<amp-img>`]({{g.doc('/content/amp-dev/documentation/components/reference/amp-img.md', locale=doc.locale).url.path}}). The `<amp-img>` tag requires a defined width and height and supports [AMP's layout system]
```

Comment:
Hey hey! Thanks for fixing these! One last nit, since this is an email doc, the links should also send readers to the filtered email view. The links should look something like this:

`({{g.doc('/content/amp-dev/documentation/components/reference/amp-img.md', locale=doc.locale).url.path}}?format=email)`

They just need the `?=email` appended after the `}}`

Reply:
done. Thanks!